solr/solr-ref-guide/src/solrcloud-autoscaling-trigger-actions.adoc

```diff
@@ -29,12 +29,13 @@ commands which can re-balance the cluster in response to trigger events.
 The following parameters are configurable:
 
 `collections`::
-A comma-separated list of collection names. If this list is not empty then
-the computed operations will only calculate collection operations that affect
-listed collections and ignore any other collection operations for collections
+A comma-separated list of collection names. This can also be a selector on the collection property e.g. `collections: {'policy': 'my_custom_policy'}` will match all collections which use the policy named `my_customer_policy`.
```

Comment:
It's a very terse description of a new powerful feature... IMHO at least it should mention that all other collection properties are supported, the property value must be equal, the implicit AND is used when multiple props are specified, and perhaps it should give some other examples.

Reply:
Thanks. I expanded the description a bit. I added another example that I am not really happy with -- I cannot think of a use-case where something other than policy might be useful especially given the fact that values must match exactly. Suggestions welcome.

Reply:
There's a typo (name mismatch) in the policy name in the doc - 'my_custom_policy' vs. 'my_customER_policy'.

One other possible property that comes to my mind would be 'router' to separate collections with TRA / CRA .. but in this case the collection could equally well be set up with its own policy - so for now maybe there's no other property that makes sense. :)

Reply:
Thanks. I had reworded the description so the `my_custom_policy` and `my_customer_policy` are both replaced with `my_policy` everywhere. I'll keep the example here because it conveys how to use more than one key/value pair.